chef/data_bags/crowbar/migrate/network/101_add_ironic_network.rb

```diff
@@ -0,0 +1,20 @@
+def upgrade(ta, td, a, d)
+  unless a["networks"].key? "ironic"
+    a["networks"]["ironic"] = ta["networks"]["ironic"]
+  end
+
+  # copy intf1 as intf3
+  a["conduit_map"].each do |cm|
+    unless cm["conduit_list"].key? "intf3"
+      cm["conduit_list"]["intf3"] = cm["conduit_list"]["intf1"]
```

Comment:
This is exactly why I introduced `intf3` instead of using the existing conduits.

Current approach is to have ironic on a separate physical network (separate NIC on the nodes which require ironic connectivity).

In practice this means that you need some custom configuration in network.json to make it work. The `"pattern": "single/.*/.*ironic.*"` mode is an actual example of this (which I use in my tests) but it may need to be adjusted for real deployments.